### ISSUE TITLE
Question: Difference between this and Pro?

### ISSUE THREAD

**Issue by DroWnThePoor** [M-Bab/linux-kernel-amdgpu-binaries#12]
This is simply a question,
What's the difference between this amdgpu and the amdgpu-Pro?
I was told before that I'd need Pro on my Elementary OS Loki install to have HDMI audio.

**Comment by M-Bab** [comment]
amdgpu-pro is the closed source driver that adds some features to the open source amdgpu. It is basically the successor of fglrx. It provides all features (HDMI audio, freesync) that are missing in the official linux kernels' amdgpu. I didn't like this driver because it made a lot of games crash. And this driver is only available to very few distributions.

amdgpu is the modern open source driver provided by AMD for all GCN GPUs. It is included in the kernel but some features are missing. And because I missed the HDMI audio so badly I started building and publishing this kernel which is based on "amd-staging-4.x" - the bleeding edge development from AMD. Here the features from amdgpu-pro are already included and I didn't had a single graphics related crash for over a year.

Performance-wise amdgpu-pro and the amdgpu should be quite the same.

TL;DR: Yes amdgpu-pro is the easiest way to get HDMI audio if your OS supports it. The other option of building your own amd-staging based kernel or install a kernel of this repo is often missed in online FAQs.
